# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/mine/hotfix-man.patch
# Copyright (C) 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- mine-0.23/mine.c.orig	2005-09-30 13:34:36.000000000 +0300
+++ mine-0.23/mine.c	2025-08-16 21:54:55.840490689 +0300
@@ -122,7 +122,7 @@
 "E.g. this will create a GEM file from the mine package installed in /:\n"
 "\n"
 "      mine -T /var/adm / mine mine.tar.bz2\n"
-"      mine -C /var/adm mine.tar.bz2 mine.gem\n"
+"      mine -C /var/adm mine.tar.bz2 mine mine.gem\n"
 "\n"
 "Hint: After installing, updating or removing packages you should run the\n"
 "      scripts 'cron.run' and 'postinstall'. This updates some important\n"
